src/components/CodeView/index.spec.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import styles from './styles.module.scss';
+
+import CodeView from '.';
+
+describe(__filename, () => {
+  const render = ({
+    content = 'some content',
+    mimeType = 'mime/type',
+  } = {}) => {
+    return shallow(<CodeView mimeType={mimeType} content={content} />);
+  };
+
+  it('renders plain text code when mime type is not supported', () => {
+    const root = render();
+
+    expect(root.find(`.${styles.CodeView}`)).toHaveLength(1);
+    expect(root.find(`.${styles.lineNumber}`)).toHaveLength(1);
+    expect(root.find(`.${styles.code}`)).toHaveLength(1);
+    expect(root.find(`.${styles.highlightedCode}`)).toHaveLength(1);
+    expect(root.find('.language-text')).toHaveLength(1);
+  });
+
+  it('renders highlighted code when language is supported', () => {
+    const content = '{ "foo": "bar" }';
+    const mimeType = 'application/json';
+    const root = render({ mimeType, content });
```

Comment:
This could be more concise:

```js
const root = render({ mimeType: 'application/json', content: '{ "foo": "bar" }' });
```

Reply:
those variables will actually be useful to fix your comment below.